src/components/BulkUploadListItem.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import prettyBytes from 'pretty-bytes';
+import {
+  ArrowPathIcon,
+  CheckCircleIcon,
+  ClockIcon,
+  XCircleIcon,
+} from '@heroicons/react/24/outline';
+import {
+  localizeDateTime,
+  relativizeDateTime,
+} from '../utils/datetime';
+import type { ApiBulkUpload } from '../pdc-api';
+import { EmailLink } from './EmailLink';
+import './BulkUploadListItem.css';
+
+enum BulkUploadStatuses {
+  pending = 'pending',
```

Comment:
Huh, I'm used to enum names (not values) being ALL_CAPS but I suppose if the linter doesn't care then why should we!  This is not a change request per se, just a musing.

Reply:
Ooh, actually, I do like that. Linter doesn't care but I'll change it anyway.